generate_wasm_pkg.sh

```diff
@@ -1,14 +1,6 @@
-git submodule update --init --recursive --remote
-cd Warp/extensions/warp-ipfs
-wasm-pack build --target web
-
-cd ../../..
-
-# Check if the source directory exists
-if [ -d "Warp/extensions/warp-ipfs/pkg" ]; then
-    # Move the pkg directory to the repository's root
-    mv Warp/extensions/warp-ipfs/pkg .
-    echo "Pkg folder moved successfully."
-else
-    echo "The source directory does not exist."
-fi
\ No newline at end of file
+wasm-pack build --target web --out-dir temp
```

Comment:
wasm-pack already creates the pkg folder by default, but either way, it should always exist anyway since we are checking in the `pkg/package.json` file to git.